packages/api/src/authentication/types/jwt-payload.ts

```diff
@@ -1,5 +1,4 @@
 export type JwtPayload = {
```

Comment:
It's different if you compare to serverside
```ts
export type TJWTAuthenticationPayload = jose.JWTPayload & {
  userName: string;
  email: string;
  sessionId?: string;
};
```

Reply:
We only need to set the JWT token in the request header here, I think the only jwt string will be enough